lib/typescript/render/assets/icons/chevron_left.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
I know that this duplicates the svg to the render library but I didn't want to create a real asset library and move all svg to that inside of this PR. We need another ticket to do that 😉